pkg/azurecr/azurecr.go

```diff
@@ -67,28 +67,43 @@ func (d *AzureCR) ListTags(repository string) ([]string, error) {
 	}
 
 	var tagsJSON azureCRTags
-	req, err := http.NewRequest("GET", endpoint, nil)
-	if err != nil {
-		return []string{}, microerror.Mask(err)
+	var tags []string
+	{
+		nextEndpoint := endpoint
+
+		for {
+			req, err := http.NewRequest("GET", nextEndpoint, nil)
+			if err != nil {
+				return []string{}, microerror.Mask(err)
+			}
+
+			req.Header.Set("Authorization", fmt.Sprintf("basic %s", d.token))
```

Comment:
I guess this was a problem we can do `r.registryClient.AddAuthorizationHeaders(req)`